cpp/cmake/Modules/ConfigureGoogleTest.cmake

```diff
@@ -27,11 +34,11 @@ if($ENV{TRAVIS})
         message(STATUS "Disabling Parallel CMake build on Travis")
     else()
         set(PARALLEL_BUILD --parallel)
-        message(STATUS "Using $ENV{CMAKE_BUILD_PARALLEL_LEVEL} build jobs on Travis")
+        message(STATUS "GTEST BUILD: Using $ENV{CMAKE_BUILD_PARALLEL_LEVEL} build jobs on Travis")
```

Comment:
Same as above this should match what is done in Arrow

Reply:
Yeah I am passing in `CMAKE_BUILD_PARALLEL_LEVEL` in Jenkins to speed things up

Reply:
There are other instances of files still referring to Travis so we will open a new issue/PR and handle it there